backend/src/models/User.ts

```diff
@@ -18,7 +19,34 @@ const UserSchema = new Schema<User, UserModel>({
     type: String,
     required: true,
   },
-});
+}, options);
+
+const User = mongoose.model<User, UserModel>("user", UserSchema);
+
+const StudentSchema = User.discriminator('Student',
+  new Schema({
+    // schema for student
+  }, options));
+
+const ReviewerSchema = User.discriminator('Reviewer',
+  new Schema({
+    // schema for reviewer
+  }, options));
+
+const RecommenderSchema = User.discriminator('Recommender',
```

Comment:
theres no role called recommender, its only counselor lol so you can remove this

Reply:
![image](https://user-images.githubusercontent.com/55229818/104949733-700b9480-598d-11eb-8c0c-06071e01cd8c.png)
bet

Reply:
bro idk all the mans keep shouting out random user types i was confused 😂